net/instaweb/rewriter/rewrite_options.cc

```diff
@@ -995,6 +996,11 @@ std::vector<DeprecatedOptionMap> kDeprecatedOptionNameList(
     kDeprecatedOptionNameData + arraysize(kDeprecatedOptionNameData)
 );
 
+// Will be initialized to a sorted list of headers not allowed in
+// AddResourceHeader.
+const StringPieceVector* fixed_resource_headers_ = NULL;
+bool http_header_separators_[256] = {false};
```

Comment:
Sorry -- the comment was meant to apply to both lines :)


On Thu, Dec 17, 2015 at 5:28 PM, Otto van der Schaaf <
notifications@github.com> wrote:

> In net/instaweb/rewriter/rewrite_options.cc
> <https://github.com/pagespeed/mod_pagespeed/pull/1200#discussion_r47969903>
> :
>
> > @@ -995,6 +996,11 @@ std::vector<DeprecatedOptionMap> kDeprecatedOptionNameList(
> >      kDeprecatedOptionNameData + arraysize(kDeprecatedOptionNameData)
> >  );
> >
> > +// Will be initialized to a sorted list of headers not allowed in
> > +// AddResourceHeader.
> > +const StringPieceVector* fixed_resource_headers_ = NULL;
> > +bool http_header_separators_[256] = {false};
>
> Done & pushed. Just to make clear, the line above is ok? (const
> StringPieceVector* fixed_resource_headers_)?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/pagespeed/mod_pagespeed/pull/1200/files#r47969903>.
>